action.yml

```diff
@@ -19,6 +19,9 @@ inputs:
   working-directory:
     descirption: 'Ember app directory'
     default: './'
+  message:
+    descirption: 'Comment message title that will be posted on PR'
```

Comment:
Oops, there is a typo here - but also in the option above, where you probably copied this from ;) Could you fix this, also in the other options?